web/styles/message_view_header.css

```diff
@@ -119,4 +119,22 @@
             vertical-align: middle;
         }
     }
+
+    .narrow_description_extended {
+        word-wrap: break-word;
```

Comment:
Is this instance of `word-wrap` even still necessary? Note too that MDN describes the `break-word` value as [deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#syntax). I think it's pretty clear that hyphens do the job they're intended.